crates/driver/src/domain/quote.rs

```diff
@@ -97,11 +95,19 @@ impl Order {
         )
     }
 
-    fn fake_auction(
+    async fn fake_auction(
         &self,
         gas_price: eth::GasPrice,
         weth: eth::WethAddress,
+        token_info: &infra::tokens::Fetcher,
```

Comment:
naming nit: I personally slightly prefer `tokens` (which was used in `Api::tokens` for example). I don't feel too strongly about the naming, but would more strongly suggest that the naming should consistent throughout the codebase.

Reply:
[3f5bda9](https://github.com/cowprotocol/services/pull/1770/commits/3f5bda9081801437c2320c2d89c0b41e3636c81b)